specs/protocol/jovian/exec-engine.md

```diff
@@ -113,6 +121,9 @@ The `gasUsed` must continue to be less than or equal to the block gas limit, mea
 (since the `daFootprint` must also be less than or equal to the block gas limit),
 blocks may have no more than `gasLimit/daFootprintGasScalar` total estimated DA usage.
 
+Note that this has the effect that the base fee is now directly influenced by DA usage because it is updated according
+to a block's `gasUsed` field.
```

Comment:
```suggestion
Note that, since the base fee continues to be updated according to a block's `gasUsed` field, the base fee may be directly influenced by DA usage.
```

Nit: slight ambiguity around "it is updated", and I also prefer to emphasize that the update mechanism remains unchanged / already exists.